salt/modules/smartos_imgadm.py

```diff
@@ -308,4 +415,91 @@ def vacuum(verbose=False):
         return list(result.keys())
 
 
+def sources(verbose=False):
+    '''
+    Return a list of available sources
+
```

Comment:
This new function, as well as all of the the other new functions below, should have a versionadded tag for `Fluorine`. :)

Reply:
Done